src/hotspot/share/oops/constantPool.hpp

```diff
@@ -77,6 +77,36 @@ class CPKlassSlot {
   }
 };
 
+class BSMAttributeEntry {
+  friend class ConstantPool;
+
+  u2 _bootstrap_method_index;
+  u2 _argument_count;
+
+  const u2* argument_indexes() const {
+    return reinterpret_cast<const u2*>(this + 1);
+  }
+  u2* argument_indexes() {
+    return reinterpret_cast<u2*>(this + 1);
+  }
+
```

Comment:
Remove blank lines at #92, #95, #103

Reply:
We typically keep a space before we change access modifier, let's keep that pattern in here. We can tighten up the rest of the code according to your suggestions, however.